cosmology/GravityManager.h

```diff
@@ -0,0 +1,367 @@
+#ifndef IPPL_GRAVITY_MANAGER_H
+#define IPPL_GRAVITY_MANAGER_H
+
+#include <memory>
+
+#include "GravityFieldContainer.hpp"
+#include "GravityFieldSolver.hpp"
+#include "GravityLoadBalancer.hpp"
+#include "Manager/BaseManager.h"
+#include "GravityParticleContainer.hpp"
+#include "Random/Distribution.h"
+#include "Random/InverseTransformSampling.h"
+#include "Random/NormalDistribution.h"
+#include "Random/Randn.h"
+
+using view_type = typename ippl::detail::ViewType<ippl::Vector<double, Dim>, 1>::view_type;
+
+/**
+ * @brief Manages the gravity simulation, including particles, fields, and load balancing.
+ * 
+ * @tparam T Type of the particle attribute.
+ * @tparam Dim Dimensionality of the particle space.
```

Comment:
This comment is ambiguous, as the dimensionality of the particle space would actually = no. of particles... 
I would say "Dimension of the configuration space".